invenio_communities/assets/semantic-ui/js/invenio_communities/community/featured-communities/index.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2022 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+import React from "react";
+import ReactDOM from "react-dom";
+import FeaturedCommunities from "./FeaturedCommunities";
+const featuredCommunitiesContainer = document.getElementById("communities-featured");
+const columnNumber = featuredCommunitiesContainer.dataset.columnNumber;
+const columnWidth = featuredCommunitiesContainer.dataset.columnWidth;
+
+ReactDOM.render(
+  <FeaturedCommunities columnNumber={columnNumber} columnWidth={columnWidth} />,
+  featuredCommunitiesContainer
+);
```

Comment:
missing blank line on the end of the file. Should we add eslint ?

Reply:
PR adding eslint here: https://github.com/inveniosoftware/invenio-communities/pull/837